### PR TITLE
feat: Add spec container_nginx_error_log

### PR DESCRIPTION
**Pull request by wushiqinlou**
### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:
This spec is used to add container check on one insights-plugins advisor rule

### CODE REVIEW

**Comment by wushiqinlou** [comment]
@xiangce - Please review.

**Review by xiangce — APPROVED** [review]

**Comment by xiangce** [comment]
@ryan-blakley - would you please have a look as well?

File: insights/parsers/nginx_log.py [hunk 1]
```diff
@@ -9,7 +9,8 @@
     for more usage information.
 """
 
-from .. import LogFileOutput, parser
+from insights import ContainerParser
+from insights import LogFileOutput, parser
```
Comment (ryan-blakley) [inline insights/parsers/nginx_log.py:13]: Could you change these to full path imports please?

```
from insights.core import ContainerParser, LogFileOutput
from insights.core.plugins import parser
```

File: insights/tests/parsers/test_nginx_log.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from insights.tests import context_wrap
 from insights.parsers import nginx_log
-from insights.parsers.nginx_log import NginxErrorLog
+from insights.parsers.nginx_log import NginxErrorLog, ContainerNginxErrorLog
```
Comment (ryan-blakley) [inline insights/tests/parsers/test_nginx_log.py:3]: Could you please move ContainerNginxErrorLog before NginxErrorLog so it's in alphabetical order?

**Review by ryan-blakley — CHANGES_REQUESTED** [review]

File: insights/parsers/nginx_log.py (hunk 1 above)
Comment (wushiqinlou) [reply insights/parsers/nginx_log.py]: @ryan-blakley - Updated, please check

File: insights/tests/parsers/test_nginx_log.py (hunk 2 above)
Comment (wushiqinlou) [reply insights/tests/parsers/test_nginx_log.py]: @ryan-blakley - Updated, please check

File: insights/parsers/nginx_log.py (hunk 1 above)
Comment (ryan-blakley) [reply insights/parsers/nginx_log.py]: Thank you.

**Review by ryan-blakley — APPROVED** [review]
Everything looks good to me.

**Pull request merged**